database1/dataGenerator2.js

```diff
@@ -5,39 +5,39 @@ const fs = require('fs');
 const photoLibrarySize = 101;
 let dataWriter = fs.createWriteStream(`data/primary-records.csv`);
 
-dataWriter.write('productId, name, category, price, rating, imageUrl, onSale');
-
-const max = 14000000;
+dataWriter.write('productId,name,category,price,rating,imageUrl,onSale');
```

Comment:
about eliminating the spaces? if so-- that is because when copying to the database I use comma as a delimiter and if there are spaces they don't get eliminated. There is probably a way to ignore whitespace, but this worked well to avoid ending up with ' name' for my column that I want to be 'name'